### PR TITLE
add documentatio for entity list with ajax pagination

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Entity list with ajax pagination
+================================
+
+To list the entities Open Orchestra uses the  `DataTables`_ plugin. To improve performance the pagination is in
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: the pagination uses ajax calls

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ajax.
+
+For each display of the information on the page (i.e. when paging, ordering, searching),
+the plugin makes an ajax request to the api with a number of variables to allow the required processing.
+
+The different variables useful for filtering your data (pagination, search, ordering)
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: for filtering the data

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+are described in this `documentation`_.
+
+For than the pagination with ajax to work, the api should return a facade with four information:
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: In order to make the pagination operationnal, the api [...]

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ * recordsTotal: Number total of entities
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: Total number of

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+ * recordsFiltered: Number of entities after filtering (search)
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: Maybe "Number of entities matching the filters"

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ * collectionName: The name of variable which contains the data
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: Could you be more precise on this one ? it's not very clear

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+ * data: The data to be displayed in the table. This is an array who contains your data (facade entity) after filtering (search, order, pagination)
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: This is an array conatining the data

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+For instance the json returned by the api for listing the workflow functions:
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: For instance, here comes the json returned

**Comment by alavieille** [comment]
@itkg-ngilain

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+To list the entities Open Orchestra uses the  `DataTables`_ plugin.
+To improve performance, the pagination uses ajax calls.
+
+For each display of the information on the page (i.e. when paging, ordering, searching),
+the plugin makes an ajax request to the api with a number of variables to allow the required processing.
+
+The different variables useful for filtering the data (pagination, search, ordering)
+are described in this `documentation`_.
+
+In order to make the pagination operational, the api should return a facade with four information:
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: four informations

**Comment by ngilain** [comment]
:+1: when you have fixed my last note

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
+Entity list with ajax pagination
+================================
+
+To list the entities Open Orchestra uses the  `DataTables`_ plugin.
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: You could start with a little bit of context : 

To list the entities in an ergonomic way, Open Orchestra uses the `DataTables`_ plugin.

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+To improve performance, the pagination uses ajax calls.
+
+For each display of the information on the page (i.e. when paging, ordering, searching),
+the plugin makes an ajax request to the api with a number of variables to allow the required processing.
+
+The different variables useful for filtering the data (pagination, search, ordering)
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: The differente variables use to filter the data are ...

**Comment by nicolasThal** [comment]
You could split the documentation in two parts : 
Context and usage.

File: en/developer_guide/entity_list_ajax_pagination.rst (hunk 9 above)
Comment (juchi) [inline en/developer_guide/entity_list_ajax_pagination.rst]: Information is invariable.
You can say "four pieces of information", or "four fields" which seems more appropriate in the context.

**Comment by alavieille** [comment]
@nicolasThal

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 12]
```diff
@@ -0,0 +1,51 @@
+Entity list with ajax pagination
+================================
+Context
+-------
+
+To list the entities in an ergonomic way, Open Orchestra uses the  `DataTables`_ plugin.
+To improve performance, the pagination uses ajax calls.
+
+For each display of the information on the page (i.e. when paging, ordering, searching),
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: I think the `when` is useless after the i.e.

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+the plugin makes an ajax request to the api with a number of variables to allow the required processing.
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: `with a number of parameters.` 

To allow ... is too heavy i think

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+
+Usage
+-----
+
+The different variables used to filter (pagination, search, ordering) the data
+are described in this `documentation`_.
+
+In order to make the pagination operational, the api should return a facade with four pieces of information:
```
Comment (nicolasThal) [inline en/developer_guide/entity_list_ajax_pagination.rst]: `the api should return a facade filled with 4 parameters :`

**Comment by alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1: with a little squash

**Pull request merged**